autoload/neomake/compat.vim

```diff
@@ -9,3 +9,56 @@ else
     return get(getbufvar(a:buf, ''), a:key, a:def)
   endfunction
 endif
+
+if exists('*json_decode')
+  let neomake#compat#json_true = v:true
+  let neomake#compat#json_false = v:false
+  let neomake#compat#json_null = v:null
+
+  function! neomake#compat#json_decode(json) abort
+    return json_decode(a:json)
+  endfunction
+else
+  let neomake#compat#json_true = 1
+  let neomake#compat#json_false = 0
+  let neomake#compat#json_null = {}
```

Comment:
Also, because it's a shorter way to keep an object ref vs `{'null': function('s:json_null')}`